tests/functionnal/Search.php

```diff
@@ -2241,6 +2241,887 @@ public function testMyselfSearchCriteria(array $criteria, array $expected)
+                'itilcategories_id' => $category,
+                '_actors' => [
+                    'requester' => [['itemtype' => 'Group', 'items_id' => $group_1]],
+                ]
+            ],
+
+            // Test set on supplier
+            [
+                'name' => 'Ticket supplier 1',
+                'content' => '',
+                'entities_id' => $root,
+                'itilcategories_id' => $category,
+                '_actors' => [
+                    'assign' => [['itemtype' => 'Supplier', 'items_id' => $supplier_1]],
+                ]
+            ],
+            [
+                'name' => 'Ticket supplier 2',
+                'content' => '',
+                'entities_id' => $root,
+                'itilcategories_id' => $category,
+                '_actors' => [
+                    'assign' => [['itemtype' => 'Supplier', 'items_id' => $supplier_2]],
+                ]
+            ],
+            [
+                'name' => 'Ticket supplier 1 + supplier 2',
+                'content' => '',
+                'entities_id' => $root,
+                'itilcategories_id' => $category,
+                '_actors' => [
+                    'assign' => [
+                        ['itemtype' => 'Supplier', 'items_id' => $supplier_1],
+                        ['itemtype' => 'Supplier', 'items_id' => $supplier_2],
+                    ],
+                ]
+            ],
+
+            // Test set on requester
+            [
+                'name' => 'Ticket user 1 (R)',
+                'content' => '',
+                'entities_id' => $root,
+                'itilcategories_id' => $category,
+                '_actors' => [
+                    'requester' => [['itemtype' => 'User', 'items_id' => $user_1]],
+                ]
+            ],
+            [
+                'name' => 'Ticket user 2 (R)',
+                'content' => '',
+                'entities_id' => $root,
+                'itilcategories_id' => $category,
+                '_actors' => [
+                    'requester' => [['itemtype' => 'User', 'items_id' => $user_2]],
+                ]
+            ],
+            [
+                'name' => 'Ticket user 1 (R) + user 2 (R)',
+                'content' => '',
+                'entities_id' => $root,
+                'itilcategories_id' => $category,
+                '_actors' => [
+                    'requester' => [
+                        ['itemtype' => 'User', 'items_id' => $user_1],
+                        ['itemtype' => 'User', 'items_id' => $user_2],
+                    ],
+                ]
+            ],
+            [
+                'name' => 'Ticket anonymous user (R)',
+                'content' => '',
+                'entities_id' => $root,
+                'itilcategories_id' => $category,
+                '_actors' => [
+                    // FIXME: Does not seem to work (maybe not possible from a form ?), will add user manually below
```

Comment:
Maybe adding the `use_notification => 1` value would fix this.

Reply:
👍